### DISCUSSION TITLE
No network, no command panel

### DISCUSSION THREAD

**Discussion by dohdohdog** [utmapp/UTM#5073] [UTM Help (macOS)] [answered]
I am trying to launch windows using the Windows 11 ARM file like other people, but neither Shift+F10 nor Fn+F10 opens a command panel to type in the workaround. What else can I do to open the panel? I have a 2021 MacBook pro m1 chip.

**Comment by osy** [comment, marked as answer]
You must press Shift+Fn+F10 at the same time.
